Correct version of tk a tcl to 8.6
--- a/configure
+++ b/configure
@@ -729,8 +729,8 @@
 if ($config_tk) { $tcl_include .= " -I$stock_tk_include_dir"; }
 $tcl_library      = "-L$stock_tcl_library_dir";
 if ($config_tk) { $tcl_library .= " -L$stock_tk_library_dir"; }
-$tcl_libs         = "-ltcl8.5";  
-if ($config_tk) { $tcl_libs = "-ltk8.5 -lX11 " . $tcl_libs; }
+$tcl_libs         = "-ltcl8.6";  
+if ($config_tk) { $tcl_libs = "-ltk8.6 -lX11 " . $tcl_libs; }
 
 @tcl_cc           = ();
 @tcl_cu           = ();
@@ -2425,8 +2425,8 @@
       $fltk_libs        = "-lfltk-x11 -lX11";
       $opengl_include   = $opengl_include . " -I/usr/X11R6/include";
       $opengl_dep_libs  = "-L/usr/X11R6/lib -lGL -framework Carbon -lXext -lX11";
-      $tcl_libs         = "-ltcl8.5-x11";
-      if ($config_tk) { $tcl_libs = "-ltk8.5-x11 -lX11 " . $tcl_libs; }
+      $tcl_libs         = "-ltcl8.6-x11";
+      if ($config_tk) { $tcl_libs = "-ltk8.6-x11 -lX11 " . $tcl_libs; }
       $vmd_libs         = "$tcl_libs";
     } else {
       ##
@@ -2540,8 +2540,8 @@
       ##   http://developer.apple.com/qa/qa2007/qa1567.html
       ##
       $opengl_dep_libs  .= " -dylib_file /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib:/System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib";
-      $tcl_libs         = "-ltcl8.5-x11";
-      if ($config_tk) { $tcl_libs = "-ltk8.5-x11 -lX11 " . $tcl_libs; }
+      $tcl_libs         = "-ltcl8.6-x11";
+      if ($config_tk) { $tcl_libs = "-ltk8.6-x11 -lX11 " . $tcl_libs; }
       $vmd_libs          = "$tcl_libs";
     } else {
       ##
